src/map/packets/message_standard.h

```diff
@@ -133,6 +134,21 @@ enum class MsgStd
     TrustEnmity                  = 300, // You cannot use Trust magic while having gained enmity.
     TrustSoloOrLeader            = 301, // You cannot use Trust magic unless you are solo or the party leader.
     AnErrorHasOccured            = 308, // An error has occurred.
+    LevelSyncMsg540              = 540, // Level Sync activated. Your level has been restricted to <Level>. Equipment effected by the level restriction will be adjusted accordingly. Experience...
```

Comment:
Embedding the number of the message into the human readable enum key isn't helping anything. 

If the message is: `Level Sync activated. Your level has been restricted to <Level>. Equipment effected by the level restriction will be adjusted accordingly. Experience...`

Then the key should be something like `LevelSyncActivated`, or `LevelSyncStart`.

In the eventually use case, what does this tell the reader?
```cpp
SetSyncTarget("", MsgStd::LevelSyncMsg553);
```

Nothing more than it was before. The point is the make these human readable.

Reply:
No problem, updated!